### DISCUSSION TITLE
Contacts' birthdays added without reminder, how to activate reminder?

### DISCUSSION THREAD

**Discussion by mattuschek** [FossifyOrg/General-Discussion#436] [General]
Hello,

I added the birthdays of the contacts in the Fossify Calendar.
I didn't pay any attention to it and left it set to "No reminder".

Now I can't find a way to activate a reminder for birthdays afterwards.
Was that forgotten?
Or is the option very well hidden?

![Screenshot_20240915_124542_Calendar](https://github.com/user-attachments/assets/4acb816c-a564-4283-a0a5-14ab54610134)
![Screenshot_20240915_124547_Calendar](https://github.com/user-attachments/assets/11eeb8db-2da0-43da-bdff-729b297f247e)

**Comment by Aga-C** [comment]
You have to remove all birthdays and synchronize again. The easiest way is:
1. Go to Settings → `Manage event types`.
2. Tap three dots next to birthdays and select Delete → `Permanently remove affected events`.
3. Add contact birthdays once again with reminders turned on.

**Reply by mattuschek** [reply]
Thanks for the solution.

The app needs improvement in this area.
Other apps can do this better...

**Reply by s0rau** [reply]
> You have to remove all birthdays and synchronize again. The easiest way is:
> 
>     1. Go to Settings → `Manage event types`.
> 
>     2. Tap three dots next to birthdays and select Delete → `Permanently remove affected events`.
> 
>     3. Add contact birthdays once again with reminders turned on.

I've used `Permanently remove affected events` in order to delete `birthday`.
- There's two entries one for `Birthday` and one for `Birthday date` => It recreates the two entries after quit and reopened.
(Only `birthday` is needed for me.)
- I can't set reminder at 12:00 for all birthdays
